setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import codecs
+from setuptools import setup
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding='utf-8').read()
+
+
+setup(
+    name='pytest-pyvista',
+    version='0.1.0',
+    author='PyAnsys',
```

Comment:
Should be PyVista here
```suggestion
    author='PyAnsys',
```

Reply:
Fixed, I couldn't apply your suggestion because you wrote PyAnsys again 😅

Reply:
Fixed. Thanks. I did not apply my suggestion originally.

Reply:
I fixed it in a previous commit, it is solved now 😄